### PR TITLE
dumpling: export consistency names, fix path log format and ping for new created db

### PR DESCRIPTION
**Pull request by lichunzhu**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/31064

Problem Summary:

### Check List

Tests 

- None

Side effects

- None

Documentation

- None

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: dumpling/export/sql.go [hunk 1]
```diff
@@ -876,6 +876,8 @@ func resetDBWithSessionParams(tctx *tcontext.Context, db *sql.DB, dsn string, pa
 	newDB, err := sql.Open("mysql", dsn)
 	if err == nil {
 		db.Close()
+		// ping to make sure all session parameters are set correctly
+		err = newDB.PingContext(tctx)
```
Comment (Ehco1996) [inline dumpling/export/sql.go:880]: do we need to close newDB when ping err!=nil?

File: dumpling/export/writer_util.go [hunk 2]
```diff
@@ -457,7 +456,7 @@ func buildFileWriter(tctx *tcontext.Context, s storage.ExternalStorage, fileName
 func buildInterceptFileWriter(pCtx *tcontext.Context, s storage.ExternalStorage, fileName string, compressType storage.CompressType) (storage.ExternalFileWriter, func(context.Context)) {
 	fileName += compressFileSuffix(compressType)
 	var writer storage.ExternalFileWriter
-	fullPath := path.Join(s.URI(), fileName)
+	fullPath := s.URI() + "/" + fileName
```
Comment (Ehco1996) [inline dumpling/export/writer_util.go:459]: is any wrong with `path.Join`？

**Review by Ehco1996 — COMMENTED** [review]
nice work, maybe open a linked issue about exported `consistency` in tiflow

File: dumpling/export/writer_util.go (hunk 2 above)
Comment (lichunzhu) [reply dumpling/export/writer_util.go:459]: Customers are confused about this file path in log. So I choose to keep the same with BR external storage here:

https://github.com/pingcap/tidb/blob/7309b08/br/pkg/storage/s3.go#L501

```go
	fmt.Println(path.Join("s3://bucket", "test"))
	fmt.Println("s3://bucket" + "/" + "test")
```
```bash
s3:/bucket/test
s3://bucket/test
```

**Review by Ehco1996 — APPROVED** [review]

**Comment by lichunzhu** [comment]
ping @lance6716

**Review by lance6716 — APPROVED** [review]

**Pull request merged**